.github/workflows/build_portable_linux_artifacts.yml

```diff
@@ -118,39 +125,64 @@ jobs:
           amdgpu_families: ${{ inputs.amdgpu_families }}
           package_version: ${{ inputs.package_version }}
           extra_cmake_options: ${{ inputs.extra_cmake_options }}
-          BUILD_DIR: build
         run: |
           python3 build_tools/github_actions/build_configure.py --manylinux
 
+      - name: Start memory monitoring
+        if: ${{ inputs.monitor_memory }}
+        id: memory-monitor
+        shell: bash
+        continue-on-error: true
+        env:
+          BUILD_DIR: ${{ env.BUILD_DIR }}
+          JOB_NAME: ${{ github.job }}
+          PHASE: "Build Phase"
+        run: |
+          # Run the memory monitor start script and capture PID
+          MONITOR_PID=$(bash build_tools/github_actions/start_memory_monitor.sh)
+          echo "monitor-pid=$MONITOR_PID" >> $GITHUB_OUTPUT
```

Comment:
nit: The comment here isn't adding much (the code is descriptive enough thanks to the script and variable names)